src/views/DidUpgradeExplainer/DidUpgradeExplainer.tsx

```diff
@@ -74,12 +108,82 @@ export function DidUpgradeExplainer({ identity }: Props): JSX.Element | null {
             {t('view_DidUpgradeExplainer_deposit')}
           </p>
 
+          {/* TODO: remove feature check when KILT checkout is released */}
```

Comment:
Is this comment superfluous? When removing the feature, we’ll be forced to clean this up…